app/models/skill.rb

```diff
@@ -18,4 +18,14 @@ def self.parse_skills
 
   scope :with_skill_type, ->(skill_name) { where(type_of_skill: skill_name) }
 
+  def career_skill
+    rank = self.rank.to_i
```

Comment:
Why are you casting the `self.rank`?

Please please please add separation between parenthesis and operators and numbers